RachaelNantale/stackoverflow-react
# #162890968 Users should be able to view all questions

#### What does this PR do?
Adds functionality for getting all questions

#### Description of Task to be completed?
- Fetch all questions from the API database
- Show all questions
#### How should this be manually tested?
Go to the homepage
You should be able to view questions and their answers if logged in
#### Any background context you want to provide?
You  do not have to be authenticated to view questions
#### What are the relevant pivotal tracker stories?
[#162890968](https://www.pivotaltracker.com/story/show/162890968)
